### PR TITLE
Drop rows and columns

### PR DESCRIPTION
**Pull request by pburnsdata**
# Issues
#19

### CODE REVIEW

File: R/trimData.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  
+  if (!is.null(settings[['baseline']][['value_col']])) {
+    data_subset %<>%
+    filter_at(settings[['baseline']][['value_col']], all_vars(. %in% settings[['baseline']][['values']]))
+  }
+   
+  if (!is.null(settings[['analysisFlag']][['value_col']])) {
+    data_subset %<>%
+    filter_at(settings[['analysisFlag']][['value_col']], all_vars(. %in% settings[['analysisFlag']][['values']]))
+  }
```
Comment (bzkrouse) [inline R/trimData.R]: 1) wouldn't the code just be ```data_subset %>% filter(settings[['analysisFlag']][['value_col']] %in% settings[['analysisFlag']][['values']])```?

2) I believe for the baseline values part, we don't necessarily want to restrict to ONLY baseline values.  We just want to make sure they are included. So I think we want:

```data_subset %>% 
    filter(settings[['analysisFlag']][['value_col']] %in% settings[['analysisFlag']][['values']] |
             settings[['baseline']][['value_col']] %in% settings[['baseline']][['values']])
```

Comment (jwildfire) [reply R/trimData.R]: agreed. something like: 

```
data_subset %<>% 
mutate(isBaseline = settings[['baseline']][['value_col']] %in% settings[['baseline']][["values]]) %>%
mutate(isAnly = settings[['analysisFlag']][['value_col']] %in% settings[['analysisFlag']][["values]]) %>%
filter(isBaseline || isAnly)
```

Comment (jwildfire) [reply R/trimData.R]: might need to throw some ifelse(is.null()) logic in those mutates too.

File: inst/eDISH_app/modules/dataUpload/dataUpload.R [hunk 2]
```diff
@@ -1,7 +1,10 @@
 dataUpload <- function(input, output, session){
   
-  ns <- session$ns
+  maxFileSize <- 20
```
Comment (jwildfire) [inline inst/eDISH_app/modules/dataUpload/dataUpload.R]: I was thinking we'd handle this in `safetyGraphicsApp()`. That we could make `maxFileSize` a new option that the user can configure for a given session.

File: R/trimData.R [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+#' @importFrom dplyr filter_at
+#' @importFrom purrr map 
+#' @importFrom magrittr "%<>%"
+#' 
+#' @keywords internal
+
+
+trimData <- function(data, settings){
+  
+  #remove columns not in settings
```
Comment (jwildfire) [inline R/trimData.R]: Nicely done!

**Pull request merged**